tomviz/PipelineModel.cxx

```diff
@@ -300,7 +336,7 @@ QVariant PipelineModel::data(const QModelIndex& index, int role) const
 
   // Data source
   if (dataSource) {
-    if (index.column() == 0) {
+    if (index.column() == 1) {
```

Comment:
It might be easier and more readable if we upgrade these to const ints or similar internally so that they can be changed, and the meaning is clearer, i.e.

namespace {
const int stateColumn;
const int labelColumn;
}